components/automate-ui/src/app/modules/infra-proxy/clients/clients.component.scss

```diff
@@ -32,6 +32,7 @@ chef-loading-spinner {
   margin: 100px auto;
   width: 100px;
   z-index: 199;
+  height: 100%;
```

Comment:
What is this doing? (That is, there is no way to review this change without some context.)

Reply:
@msorens yes, Actually we have added the pagination on the list for that we have added this height here.